lib/parallel_tests/cli.rb

```diff
@@ -135,8 +135,10 @@ def report_failure_rerun_commmand(test_results, options)
         puts "\n\nTests have failed for a parallel_test group. Use the following command to run the group again:\n\n"
         failing_sets.each do |failing_set|
           command = failing_set[:command]
+          env = failing_set[:env]
           command = @runner.command_with_seed(command, failing_set[:seed]) if failing_set[:seed]
-          puts Shellwords.shelljoin(command)
+          env_str = "TEST_ENV_NUMBER=#{env['TEST_ENV_NUMBER']} PARALLEL_TEST_GROUPS=#{env['PARALLEL_TEST_GROUPS']}"
```

Comment:
```suggestion
          env_str = ["TEST_ENV_NUMBER", "PARALLEL_TEST_GROUPS"].map { |e| "#{e}=#{env[e]}" }.join(" ")
```

Reply:
... do we even need PARALLEL_TEST_GROUPS ?
I think the reason for this PR was that the re-run did not hit the same db or something like that ?

Reply:
I guess yes. Will be useful when `--only-group` is used. Otherwise, it will be difficult to differentiate commands.